tests/Integration/Database/WipeCommandTest.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+namespace Illuminate\Tests\Integration\Database;
+
+use Illuminate\Database\QueryException;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Schema;
+use Orchestra\Testbench\TestCase;
+
+class WipeCommandTest extends TestCase
```

Comment:
This should extend `Illuminate\Tests\Integration\DatabaseTestCase` and the default connection can either be MySQL, Postgres, MSSQL or SQLite based on GitHub Actions (`databases.yml`)

Reply:
@crynobone Thank you, I made the correction.